pyproject.toml

```diff
@@ -25,7 +25,7 @@ packages = [
 [tool.poetry.dependencies]
 python = ">=3.8,<4.0"
 importlib-metadata = {version = "^4.0", python = "<3.8"}
-ansys-fluent-core = "~=0.18.dev0"
+ansys-fluent-core = ">=0.18.0"
```

Comment:
I think this is more appropriate now as we have now different release cadence for pyfluent and pyfluent-parametric. The pyfluent-parametric 0.9 will remain compatible with pyfluent 0.18.0, 0.18.1, 0.19.0, ... so on. After a pyfluent release in future, if pyfluent-parametric becomes incompatible we need to fix the incompatibility and make a new public release of parametric with the upgraded version of pyfluent dependency.

Reply:
Sounds good to me, both `">=0.18"` and `"~=0.18"` (more restrictive) should work fine in my understanding.

`">=0.18"` might require less maintenance and work from our part, as long as future pyfluent core releases don't break pyfluent-parametric. `"~=0.18"` is safer.

Reply:
Just for my understanding, what is the difference between `>=0.18` and `~=0.18`? In the [pep](https://peps.python.org/pep-0440/#compatible-release), it is mentioned these are "approximately" equivalent, not sure in which scenarios these will be different.

Reply:
In my understanding, this is how it works:
- `>=0.18` means that all versions including 0.18.0 and above are valid
- `~=0.18` is the same as `>=0.18` with `==0.18.*`, which is the same as saying `>=0.18` with `<0.19`, or that all versions above 0.18.0 but before 0.19.0 are valid

But I haven't tested this using version specifiers as I remember that you did in the past. Are you noticing inconsistent behavior?

Reply:
As per my testing, both specifiers have the same behaviour, so I was wondering what the difference could be.

Reply:
OK, so indeed the version specifiers implementation seems to not fully agree with the PEP documentation (or I am misunderstanding it), or it may not be fully valid for pre-release versions starting with `0.*`

What I was talking about above was `">=0.18"` compared to `"~=0.18.0"` (and not `"~=0.18"`), going to edit that comment so it is accurate

Reply:
I was not considering the major release, for which `>=0.18` and `~=0.18`. We need compatibility for all future minor releases. so we are good here. Thanks for clarifying this.

Reply:
Sorry for the confusion, made a few edits above! Please refresh and take a look again. 

And I think I would prefer using `>=0.18` rather than `~=0.18` if we don't want to have to update manually once 0.19 and above comes out, as mentioned in my other suggestion below, as I believe that is less confusing

Reply:
By the way, there is still something wrong with these specifiers, according to the error message this should not be true (this was the reason for the error to begin with):
![image](https://github.com/ansys/pyfluent-parametric/assets/132297401/a229f07d-dd4a-47ca-899a-336998235603)

Very odd